base/audit_events.go

```diff
@@ -72,6 +240,638 @@ var AuditEvents = events{
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: true,
+		EventType:          eventTypeData,
+	},
+	AuditIDReplicationConnect: {
+		Name:        "Replication connect",
+		Description: "A client connected using WebSocket/BLIP for replication",
+		MandatoryFields: AuditFields{
+			"db":  "database name",
+			"cid": "context id",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: true,
+		EventType:          eventTypeUser,
+	},
+	AuditIDReplicationDisconnect: {
+		Name:        "Replication disconnect",
+		Description: "A client disconnected from WebSocket/BLIP replication",
+		MandatoryFields: AuditFields{
+			"db":  "database name",
+			"cid": "context id",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: true,
+		EventType:          eventTypeUser,
+	},
+	AuditIDISGRCreate: {
+		Name:        "Create Inter-Sync Gateway Replication",
+		Description: "A new Inter-Sync Gateway Replication was created",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
+			"config":         "JSON representation of replication config",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDISGRRead: {
+		Name:        "Read Inter-Sync Gateway Replication",
+		Description: "Information about this Inter-Sync Gateway Replication was viewed",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDISGRUpdate: {
+		Name:        "Update Inter-Sync Gateway Replication",
+		Description: "Inter-Sync Gateway Replication was updated",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
+			"config":         "JSON representation of new replication config",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+	},
+	AuditIDISGRDelete: {
+		Name:        "Delete Inter-Sync Gateway Replication",
+		Description: "Inter-Sync Gateway Replication was deleted",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDISGRStatus: {
+		Name:        "Inter-Sync Gateway Replication status",
+		Description: "Inter-Sync Gateway Replication status was document viewed",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
+		},
+		EnabledByDefault:   false,
+		FilteringPermitted: true,
+	},
+	AuditIDISGRStart: {
+		Name:        "Inter-Sync Gateway Replication start",
+		Description: "Inter-Sync Gateway Replication was started",
+		MandatoryFields: AuditFields{
+			"db":             "database name",
+			"replication_id": "replication id",
```

Comment:
This will have a `cid` from the http request and also different `cid` from the replication itself.

Reply:
Given the replication isn't necessarily going to be running on the node that the started request came in on, we cannot determine blip context id.

This is another one of those cases where we want to audit the action the user performed and not the result of the action, if that makes sense?